### ISSUE TITLE
Multiplayer Desync from Constructible Siege Equipment

### ISSUE THREAD

**Issue by augustus8825** [JustusAvramenko/delenda_est#65]
_I originally posted this on the mod.io comments but realized it probably isn't the best way to communicate bugs, so I'll just forward it here:_

It looks like any time any constructible siege equipment (Catapults, Siege Towers, etc) are placed, or even just selected from the construction toolbar, it desyncs multiplayer games. This happens consistently in both the release version and dev-build dedc32e from November 8th, 2024.

**Comment by JustusAvramenko** [comment]
You are right. It's a bug in the base game that is getting worked on, because the base game wants to add constructible Onagers for Roman soldiers. It's not a trivial fix, so for A27 I may have to temporarily remove the constructible siege equipment and make them trainable from Arsenals again until Release 28 when the fix will likely have been committed.
